packages/terra-aggregator/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-aggregator",
+  "main": "lib/Aggregator.js",
+  "version": "0.0.0",
+  "description": "The Aggregator provides focus-based mechanisms for managing actions across disparate components.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-aggregator",
+    "Aggregator",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-app-delegate": "^1.10.0",
+    "terra-button": "^1.14.0",
+    "terra-list": "^1.19.0",
+    "terra-content-container": "^1.15.0",
+    "terra-clinical-header": "^1.7.0",
+    "terra-clinical-action-header": "^1.6.0",
+    "terra-form": "^1.16.0",
```

Comment:
Looks like some of these should be devDependencies

Reply:
There a ton of weird dependencies now due to how the generated site works. As far as I know, devDependencies are not sufficient for the time being.

Some of these will go away as I create more packages and move the examples around.

Reply:
Should there be an issue for this? We don't want to bring in dependencies only needed in the generated site.

Reply:
We release the tests as runnable artifacts now. This won't affect your usage of the aggregator or pull more assets into your page, It just may make your node_modules folder larger.